features/step_definitions/work_download_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Then /^I should receive a file of type "(.*?)"$/ do |filetype|
-  mime_type = filetype == "azw3" ? "application/x-mobi8-ebook" : MIME::Types.type_for("foo.#{filetype}").first
+  mime_type = filetype == "azw3" ? "application/x-mobi8-ebook" : Marcel::MimeType.for("foo.#{filetype}").to_s
```

Comment:
I _think_ these can be simplified too

Reply:
This one I'm not sure, I had to use extend method (for registering the type) since it's not natively on the extensive list of Marcel. Buuut I'll check anyway, thanks for the help! I'll change it tonight